Doppler.UsersApi/Infrastructure/AccountRepository.cs

```diff
@@ -0,0 +1,29 @@
+using Dapper;
+using Doppler.UsersApi.Model;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Doppler.UsersApi.Infrastructure
+{
+    public class AccountRepository : IAccountRepository
+    {
+        private readonly IDatabaseConnectionFactory _connectionFactory;
+        public AccountRepository(IDatabaseConnectionFactory connectionFactory)
+        {
+            _connectionFactory = connectionFactory;
+        }
+        public async Task<ContactInformation> GetContactInformation(string email)
+        {
+            using (IDbConnection connection = await _connectionFactory.GetConnection())
+            {
+                var results = await connection.QueryAsync<ContactInformation>(@"
+                    SELECT  U.FirstName, U.LastName, U.Email, U.IdIndustry AS Industry, U.Company, U.PhoneNumber AS Phone, U.Address, U.ZipCode, CI.Name AS City, U.CityName, S.Name AS Province, CO.Code AS Country
+                    FROM    [User] U INNER JOIN [State] S ON U.IdState = S.IdState AND U.IdBillingState = S.IdState INNER JOIN City CI ON S.IdState = CI.IdState INNER JOIN [Country] CO ON S.IdCountry = CO.IdCountry
+                    WHERE   U.Email = @email",
+                    new { email });
```

Comment:
Muy, muy, muy menor y discutible, a mi me parece que este formato sería mejor:

```suggestion
                var results = await connection.QueryAsync<ContactInformation>(@"
SELECT 
    U.FirstName, 
    U.LastName, 
    U.Email, 
    U.IdIndustry AS Industry, 
    U.Company, 
    U.PhoneNumber AS Phone, 
    U.Address, 
    U.ZipCode, 
    CI.Name AS City, 
    U.CityName, 
    S.Name AS Province, 
    CO.Code AS Country
FROM
    [User] U 
    INNER JOIN [State] S ON U.IdState = S.IdState AND U.IdBillingState = S.IdState
    INNER JOIN City CI ON S.IdState = CI.IdState
    INNER JOIN [Country] CO ON S.IdCountry = CO.IdCountry
WHERE
    U.Email = @email",
                    new { email });
```

Reply:
es mas claro leerlo, lo cambio